components/blocks/VerticalCards.tsx

```diff
@@ -0,0 +1,63 @@
+import Link from 'next/link';
+import React from 'react';
+import css from 'styled-jsx/css';
+import Globe from './zzGlobe'; // Import the Globe component
+
+const Card = ({ cardItem }) => (
+  <Link href={cardItem.link || "#"}>
+    <div className="relative p-6 m-4 rounded-md group flex flex-col lg:flex-row bg-gradient-to-br from-white/25 via-white/50 to-white/75 break-inside-avoid rounded-xl shadow-md transform transition-transform duration-300 hover:scale-105 transform-origin-center">
+      <div className="w-full lg:w-1/3 lg:h-full mb-4 lg:mb-0 mr-0 lg:mr-6">
+        <img src={cardItem.image} alt={cardItem.headline} className="object-cover w-full h-full" />
+      </div>
+      <div className="flex-grow flex flex-col">
+        <h2 className="font-bold text-3xl mb-2">{cardItem.headline}</h2>
```

Comment:
Looking at the headings hierarchy, I think this should be a h3

Reply:
✅